deploy/dashboard.yaml

```diff
@@ -50,9 +61,10 @@ spec:
       labels:
         app: dapr-dashboard
     spec:
+      serviceAccountName: dashboard-reader
       containers:
-      - name: dapr-dashboard
-        image: docker.io/daprio/dashboard:0.2.0
-        ports:
-        - containerPort: 8080
-        imagePullPolicy: Always
+        - name: dapr-dashboard
+          image: docker.io/daprio/dashboard:0.2.0
```

Comment:
it is still 0.1.0  as 0.2.0 is not yet there released ?

Reply:
True... I'll fix that.